src/GraphQL/Execution/ExecutionNode.cs

```diff
@@ -88,25 +88,25 @@ public IEnumerable<string> Path
             }
         }
 
-        private static string GetStringIndex(int index) => index switch
-        {
-            0 => "0",
-            1 => "1",
-            2 => "2",
-            3 => "3",
-            4 => "4",
-            5 => "5",
-            6 => "6",
-            7 => "7",
-            8 => "8",
-            9 => "9",
-            10 => "10",
-            11 => "11",
-            12 => "12",
-            13 => "13",
-            14 => "14",
-            15 => "15",
-            _ => index.ToString()
+        private static object GetObjectIndex(int index) => index switch
```

Comment:
Well :) This method is no longer needed.

Reply:
No. I was wrong. We need it to eliminate boxing. You need to change implementation to return cached boxed integers instead of boxing them every call.

Reply:
I know the goal but wasn't sure what was necessary to accomplish it.  I had `1 => (object)1`, but C# reported that the cast to object was unnecessary, which is why it is currently `1 => 1`.  I thought of using an array, but wasn't sure if that would be slower or faster than a switch expression.  I'm not going to spend time benchmarking this (it never executes anyway unless you're logging), but if you have a suggestion I'll code it.  Should I create 16 readonly fields?  e.g. `private static readonly object num0 = 0;`

Reply:
Right.

Reply:
Done